pynamodb/models.py

```diff
@@ -17,6 +17,7 @@
 from pynamodb.types import HASH, RANGE
 from pynamodb.compat import NullHandler
 from pynamodb.indexes import Index, GlobalSecondaryIndex
+from pynamodb.settings import RequestSessionWithHeaders, RETRY_ENABLED
```

Comment:
i'm not sure what's happening here

to recap, the user should be able to put a settings file in a known location on the system, e.g. `/etc/pynamodb/settings.py`

we need to check for existence of that file. if it exists, we load it. it should have the same parameters as the Meta class. if the parameter exists in the config file and not in the Meta class, then we use the parameter from the config file as the default. if the parameter exists in the Meta class, and the config file, the value in the Meta class takes precedence over the value in the config file.